docs/Current_Code.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python
```

Comment:
code shouldn't be in the docs directory but in the ``pytentiostat/pytentiostat`` directory.

We should choose a good name for this python Module.  Something like ``arduinobase.py`` because it contains your arduino base class.